ome-dundeeomero.yml

```diff
@@ -245,29 +256,38 @@
     #omero_server_db_dumpdir_parent: /tmp
     #omero_server_db_dumpdir_name: nightly-pg_dump_omero.dir
 
-    ice_version: "3.6"
+    nginx_version: 1.16.1
     postgresql_version: "9.6"
     filesystem: "xfs"
-    omero_figure_release: "{{ omero_figure_release_override | default('v4.0.2') }}"
-    omero_server_config_set:
+    omero_figure_release: "{{ omero_figure_release_override | default('4.2.0') }}"
+
+    omero_web_config_set_production:
```

Comment:
This surprises me for "omero.db.poolsize"

Reply:
It's what's currently there, can you open a new PR after this is merged with whatever you think is appropriate?

Reply:
It's less the value and more the fact that it's set on `omero_web_*` at all. Why did this change from `omero_server` to `omero_web`?

Reply:
Sorry, it's a typo. Fixed.

Reply:
:+1: does the playbook now show a diff, because if so, we may need to re-deploy ASAP.

Reply:
It shouldn't, the var is only used indirectly via it's reference